### PR TITLE
Remove `boxartguide` command - Replaced with sending `boxart` with no arguments

### PR DESCRIPTION
**Pull request by YourKalamity**

### CODE REVIEW

**Comment by Epicpkmn11** [comment]
Might be good to do the same thing for `twlsfx` (convert)/`twlbgm` (wiki) and `unlaunch` (convert)/`unlaunchbg` (wiki)

File: cogs/convert.py [hunk 1]
```diff
@@ -186,6 +193,14 @@ async def unlaunch_background(self, ctx, *args):
         """
         Convert an attachment or linked image to an Unlaunch GIF file
```
Comment (LightSage) [inline cogs/convert.py:194]: It would be nice if the description got changed to reflect the changes made.

File: cogs/convert.py [hunk 2]
```diff
@@ -356,6 +374,14 @@ async def twlbgm(self, ctx, filelink=None):
         """
         Converts an attached, or linked, audio file to TWiLight Menu's BGM format
```
Comment (LightSage) [inline cogs/convert.py:375]: The same should be done here

**Review by LightSage — CHANGES_REQUESTED** [review]

**Review by lifehackerhansol — APPROVED** [review]

**Comment by NightScript370** [comment]
....I meant to approve the request, oops

**Review by NightScript370 — APPROVED** [review]

**Pull request merged**